socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
```

Comment:
It always bothers me to see a class instance get created without not necessarily being used. It adds noise and makes it read like "We create it, therefore we will need it" when that's not the case. 
I'd venture to do `storage_exception = None` (for the benefit of the last two lines) and then do something like
```python
except Exception:
    if storage_exception is None:
        storage_exception = PolyStorageError()
```